Documentation/Includes.rst.txt

```diff
@@ -1,11 +1,14 @@
-﻿.. This is 'Includes.txt'. It is included at the very top of each and
+.. This is 'Includes.txt'. It is included at the very top of each and
    every ReST source file in THIS documentation project (= manual).
+   
+   added: pn textrole, "proper name", like `TYPO3 Core Team`:pn:
```

Comment:
This sounds like a version maintenance besides Git. Maybe only add a plain comment as example.